Feat/i18n

merx88 opened the pull request:
# What i do 

## 1. Apply i18n

Apply i18n according to the app route

Files related to i18n can be found in the utils directory.

### settings.ts
It includes i18n settings

### index.ts
It is used, when components is rendered with SSR  -> you can check example in Hero.tsx

### client.ts
It is used, when components is rendered with CSR (aka. "use client") -> you can check example in GlobalFooter.tsx


## 2. i18n route(middleware.ts)

I used sub path route (ex) hebi.im/ko/#hero)


## 3. i18n JSON file builder

locales json filies and build.js can be find in locales
You can parse the Google sheet with build.js

kms0219kms commented:
@merx88 generated json file should be in .gitignore since it might create build failure (context mismatch)

merx88 commented:
> @merx88 generated json file should be in .gitignore since it might create build failure (context mismatch)

I updated gitignore and removed json files according to your comment

kms0219kms commented:
@merx88 Can you make sure in package.json, dev or build command should build the i18n file?

merx88 commented:
> @merx88 Can you make sure in package.json, dev or build command should build the i18n file?

Done!
